angular/lib/src/lib/slider/slider.component.ts

```diff
@@ -0,0 +1,385 @@
+
+			.au-slider-progress[disabled],
+			.au-slider-handle[disabled] {
+				background-color: var(--bs-dark-bg-subtle, #b3b3b3);
+				cursor: not-allowed;
+			}
+
+			.au-slider-label[disabled],
+			.au-slider-label-vertical[disabled] {
+				color: var(--bs-dark-bg-subtle, #b3b3b3);
+			}
+
+			au-slider[disabled] {
+				cursor: not-allowed;
+			}
+
+			.au-clickable-slider-area {
+				height: 1.5rem;
+				width: 100%;
+				transform: translateY(-50%);
+			}
+
+			.au-clickable-slider-area-vertical {
+				width: 1.5rem;
+				height: 100%;
+				transform: translateX(-50%);
+				flex-shrink: 0;
+			}
+		`,
+	],
+})
+export class SliderComponent implements OnChanges, AfterViewInit, OnDestroy {
+	/**
+	 * auSlider element reference
+	 */
+	private _elementRef = inject(ElementRef);
+	private _sliderDirective: ReturnType<Directive>;
+	private _zone = inject(NgZone);
+	// needed to avoid changedetection on the mouse move over the same coordinate (vertical or horizonal movement)
+	private _prevCoordinate = -1;
+
+	/**
+	 * Minimum label element reference
+	 */
+	@ViewChild('minLabel')
+	private _minLabelRef: ElementRef;
+	/**
+	 * Maximum label element reference
+	 */
+	@ViewChild('maxLabel')
+	private _maxLabelRef: ElementRef;
+
+	readonly _widget = callWidgetFactory({
+		factory: createSlider,
+		widgetName: 'slider',
+		events: {
+			onValuesChange: (event) => {
+				this.onChange(event);
+				this.onTouched();
+				this.valuesChange.emit(event);
+			},
+		},
+	});
+	/**
+	 * SliderWidget hold the state and actions applicable to the auSliderComponent
+	 */
+	readonly widget = toSlotContextWidget(this._widget);
+	readonly state: Signal<SliderState> = toSignal(this._widget.state$, {requireSync: true});
+	readonly api = this._widget.api;
+
+	/**
+	 * CSS classes to be applied on the widget main container
+	 */
+	@Input('auClassName') className: string | undefined;
+
+	/**
+	 * Minimum value that can be assigned to the slider
+	 */
+	@Input('auMin')
+	min: number | undefined;
+
+	/**
+	 * Maximum value that can be assigned to the slider
+	 */
+	@Input('auMax')
+	max: number | undefined;
+
+	/**
+	 * Unit value between slider steps
+	 */
+	@Input('auStepSize')
+	stepSize: number | undefined;
+
+	/**
+	 * Current slider values
+	 */
+	@Input('auValues')
+	values: number[] | undefined;
+
+	/**
+	 * If `true` slider value cannot be changed but the slider is still focusable
+	 */
+	@Input('auReadonly')
+	readonly: boolean | undefined;
+
+	/**
+	 * If `true` slider value cannot be changed and the slider cannot be focused
+	 */
+	@Input('auDisabled')
+	disabled: boolean | undefined;
+
+	/**
+	 * If `true` is vertically positioned otherwise it is horizontal
+	 */
+	@Input('auVertical')
+	vertical: boolean | undefined;
+
+	/**
+	 * An event emitted when slider values are changed
+	 *
+	 * Event payload equals to the updated slider values
+	 */
+	// TODO remove the disable when angular-check-props fixed
+	// eslint-disable-next-line @agnos-ui/angular-check-props
```

Comment:
I don't see any issue when removing this `eslint-disable-next-line` line.

Reply:
hmm, did the npm install and build and it underlines it saying that it is not overridden in the constructor, mb the VS targets the wrong thing again